prowler/__main__.py

```diff
@@ -305,7 +311,14 @@ def prowler():
             global_provider.output_options,
         )
 
-        if findings:
+        if (
```

Comment:
Please include a comment here to know the reason behind that or create a new attribute based on that conditional not to repeat it, up to you.

Reply:
Done!